scipy/optimize/_constraints.py

```diff
@@ -249,6 +250,29 @@ def __init__(self, constraint, x0, sparse_jacobian=None,
         self.bounds = (lb, ub)
         self.keep_feasible = keep_feasible
 
+    def violation(self, x):
```

Comment:
I think this function can be removed now, right? Since I merged the cross-linked pre-requisite PR recently